<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="showImg">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>865</width>
      <height>485</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="chooseFuncWidget">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>10</y>
      <width>300</width>
      <height>490</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="funcCamera">
     <attribute name="title">
      <string>基于摄像头的识别</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>260</width>
        <height>121</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QPushButton" name="pBtnC_CameraSwitch">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>打开摄像头</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pBtnC_CaptureFrame">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>10</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>捕获帧</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pBtnC_ConductDetection">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>执行人脸检测</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pBtnC_MethodSwitch">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>转化为机器学习方法</string>
       </property>
      </widget>
     </widget>
     <widget class="QScrollArea" name="sArea_CameraCapture">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>260</width>
        <height>300</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>300</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>248</width>
         <height>600</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>600</height>
        </size>
       </property>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>10</y>
          <width>240</width>
          <height>180</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>调整摄像头窗口分辨率：</string>
        </property>
        <widget class="QFrame" name="frame_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>220</width>
           <height>120</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <widget class="QRadioButton" name="radBtnR_ResolutionRatio1280x720">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1280x720</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="radBtnR_ResolutionRatio1440x810">
          <property name="geometry">
           <rect>
            <x>135</x>
            <y>10</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1440x810</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radBtnR_ResolutionRatio1600x900">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1600x900</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radBtnR_ResolutionRatio1760x990">
          <property name="geometry">
           <rect>
            <x>135</x>
            <y>40</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1760x990</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radBtnR_ResolutionRatio1920x1080">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>90</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1920x1080</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>200</y>
          <width>240</width>
          <height>180</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>边缘检测算子选择:</string>
        </property>
        <widget class="QFrame" name="frame_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>220</width>
           <height>120</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <widget class="QRadioButton" name="radBtnE_edgeDetection_Roberts">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Roberts算子</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radBtnE_edgeDetection_Sobel">
          <property name="geometry">
           <rect>
            <x>135</x>
            <y>10</y>
            <width>73</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Sobel算子</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="radBtnE_edgeDetection_Prewitt">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>73</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Prewitt算子</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radBtnE_edgeDetection_Laplacian">
          <property name="geometry">
           <rect>
            <x>135</x>
            <y>90</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Laplacian算子</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>390</y>
          <width>240</width>
          <height>180</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>阈值分割算法选择：</string>
        </property>
        <widget class="QFrame" name="frame_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>220</width>
           <height>120</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <widget class="QRadioButton" name="radBtnI_ImageSplit_DoubleHill">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>直方图双峰</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radBtnI_ImageSplit_AdaptiveThreshold">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>45</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>自适应阈值</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="radBtnI_ImageSplit_Iteration">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>80</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>迭代阈值</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="funcProduce">
     <attribute name="title">
      <string>对人脸区域进行处理</string>
     </attribute>
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>280</width>
        <height>120</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>240</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>请选择一个ROI图像并进行处理！</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pBtn_ChooseROI">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>选择ROI图像</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>280</width>
        <height>140</height>
       </rect>
      </property>
      <property name="title">
       <string>图像处理</string>
      </property>
      <widget class="QPushButton" name="pBtn_FindOrgans">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>检测五官</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pBtn_CartoonType">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>漫画风处理</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pBtn_FunhouseType">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>哈哈镜处理</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pBtn_AddEdge">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>80</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>加边框处理</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="File">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actF_OpenFile"/>
    <addaction name="separator"/>
    <addaction name="actF_SaveFile"/>
    <addaction name="actF_SaveFileAsCopy"/>
    <addaction name="actF_Reload"/>
   </widget>
   <widget class="QMenu" name="Help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actH_Greetings"/>
   </widget>
   <addaction name="File"/>
   <addaction name="Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actF_OpenFile">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actF_SaveFile">
   <property name="text">
    <string>保存文件</string>
   </property>
  </action>
  <action name="actF_SaveFileAsCopy">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actF_Reload">
   <property name="text">
    <string>重载文件</string>
   </property>
  </action>
  <action name="actH_Greetings">
   <property name="text">
    <string>感谢使用</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
